src/test/tests/quickrecipes/colortables.py

```diff
@@ -19,6 +21,27 @@
 # comments of the form '# sometext {' and '# sometext }' bracket the sections
 # that are 'literalinclude'd in quickrecipes.rst.
 
+def introspectingColorTable():
```

Comment:
Its code that is published in the QR docs and so needs to be tested to ensure it actually works. It was the only code in the color tables section that was not `literalinclude`d code. I choose NOT to surround it with the other logic I used for all the other stuff I *added* because all that other stuff follows a new and improved paradigm using the `vqr_utils.py`, wrapped with exception checking and with error string checking but don't create pesky *baseline* files we then need to constantly manage separately. This was so that it was consistent with all the other methods in `colortables.py`

Honestly, there are a lot of cases of quick recipe code snipits that simply demonstrate how to do something but don't actually do any testing work to confirm those operations did what was intended or what they were supposed to do.